src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -139,6 +139,9 @@ method newMessagesLoaded*(self: Module, messages: seq[MessageDto], reactions: se
           item.pinned = true
           item.pinnedBy = p.pinnedBy
 
+      if m.editedAt != 0:
```

Comment:
@Khushboo-dev-cpp I would rather use this ">" instead of "!=", cause in some languages "0" is not the default value for uninitialized int variable and that may cause some hidden issues, but since that's not a case in Go (there is "0" used as default int value) you are free to decide whether to update this or not. :)

Reply:
kept it is > than I am okay with either